pom.xml

```diff
@@ -760,8 +760,6 @@
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcpkix-jdk15on</artifactId>
         <version>1.69</version>
-        <scope>compile</scope>
-        <optional>true</optional>
```

Comment:
I think this is not correct... this should still be optional otherwise we have it as non optional dependency for handler as well. 

Same its true for the other change in here.